README.md

```diff
@@ -14,6 +14,8 @@ In this GitHub repository, the following SuiteCloud SDK tools are available:
     <img src="https://img.shields.io/npm/v/@oracle/suitecloud-unit-testing.svg" alt="npm-unit-testing"/>
 </a>
 
+- [SuiteCloud Extension for Visual Studio Code](./packages/vscode-extension)
```

Comment:
Hi David, would you like to merge this PR now or after all 21.2 doc updates are complete?